BUILD

```diff
@@ -5,54 +5,160 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_bundle")
 container_bundle(
     name = "all",
     images = {
-        "gcr.io/{PROJECT_ID}/base:latest": "//base:base_root_amd64_debian9",
-        "gcr.io/{PROJECT_ID}/base:nonroot": "//base:base_nonroot_amd64_debian9",
-        "gcr.io/{PROJECT_ID}/base:debug": "//base:debug_root_amd64_debian9",
-        "gcr.io/{PROJECT_ID}/base:debug-nonroot": "//base:debug_nonroot_amd64_debian9",
+        "gcr.io/{PROJECT_ID}/static:{COMMIT_SHA}": "//base:static_root_amd64_debian9",
```

Comment:
I think that the vast majority of this could be computed using a "dictionary comprehension", see: https://docs.bazel.build/versions/master/skylark/lib/dict.html

That would seriously boost my confidence that we've covered everything (and would probably fill in some gaps we've had previously).

I think that the main pieces it wouldn't "just work" for would be the aliases we submit as sugar, but I think enumerating those explicitly would be very nice.

Reply:
All told, it will probably also be shorter!

Reply:
Yeah I thought about going that route in this PR but I thought it might make a better follow-up since we were already kind of unsure about the list being out of sync. Seeing the whole list explicitly spelled out is easier to compare to the previous list.

Let me know if you'd prefer this to be in this PR and I can hack on it, otherwise it'll be a follow-up.

Reply:
Yeah, after hacking on this for a few minutes I'm not sure it's cleaner to use dict comprehensions, versus just spelling it all out. For one, it [doesn't seem possible to add two dicts](https://github.com/bazelbuild/skydoc/issues/10), and [`update` returns `None`](https://docs.bazel.build/versions/master/skylark/lib/dict.html#update). 😖 

Maybe I'm just not good enough at Skylark to make it happen. If people have better luck making this expressive I'll gladly defer to them. I agree it probably would help make sure we cover (and keep covering!) all the combinations. In the meantime I think this is an improvement we can build on in future changes.